website/src/app/api.service.ts

```diff
@@ -0,0 +1,73 @@
+import { Injectable } from '@angular/core';
+import { HttpModule, Http } from '@angular/http';
+import 'rxjs/Rx';
+import { Observable } from 'rxjs/Observable';
+import { User } from './user'
+import { environment } from 'src/environments/environment';
+
+const API_URL = environment.apiUrl;
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApiService {
```

Comment:
Name is too generic. If besides users you had i.e. rabbits and birds on the back-end you'd either put it all here or needed to rename.